utils/step.py

```diff
@@ -24,17 +10,19 @@ class Block:
     definition: list[list[int]] = field(default_factory=list)
 
 @dataclass
-class Gcode:
+class Vcfg:
     name: str
-    file: str
+    technique: str
+    configs: dict
+
 
-class Experiment:
-    def __init__(self, solution, block, technique, vcfg, gcode):
-        self.solution: str = solution
-        self.block: Block = block
-        self.technique: str = technique
-        self.vcfg: CV = vcfg
-        self.gcode: Gcode = gcode
+class Step:
```

Comment:
```suggestion
class ExperimentStep:
```
and make that a global change  - the class `Step` is just a bit too vague once it gets into application code